homeassistant/components/homeassistant_hardware/firmware_config_flow.py

```diff
@@ -215,6 +280,14 @@ async def async_step_addon_operation_failed(
             },
         )
 
+    async def async_step_pre_confirm_zigbee(
```

Comment:
This seems odd, how do other integrations do it?

Reply:
This shouldn't be necessary.